project/AkkaDisciplinePlugin.scala

```diff
@@ -66,14 +70,14 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
   /**
     * Remain visibly filtered for future code quality work and removing.
     */
-  val undisciplineScalacOptions = Seq(
+  val undisciplineScalacOptions = Set(
```

Comment:
why change? isn't it anyway converted to a Seq?

Reply:
to use `--`, could do filter

Reply:
I have no opinion